src/go-chaincode/deploy.sh

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/env bash
 
 # Go chaincode specific deploy script
+
+# shellcheck source=src/common/env.sh
+source "${SCRIPT_DIR}/common/env.sh"
+# shellcheck source=src/common/utils.sh
+source "${SCRIPT_DIR}/common/utils.sh"
+# shellcheck source=src/common/blockchain.sh
+source "${SCRIPT_DIR}/common/blockchain.sh"
+
+if [[ ! -f $CONFIGPATH ]]; then
+  echo "No deploy configuration at specified path: ${CONFIGPATH}"
+  exit 1
+fi
+
+install_jq
+setup_service_constants
+provision_blockchain
+parse_fabric_config $CONFIGPATH
```

Comment:
I do not like the fact that basically the meet of the deploy capabilities have been abstracted within a function called "parse_fabri_config"  .  I also do not like that this function has been placed on a common library even though the logic there is unlikely to be used anywhere else.  I would actually bring that logic here, before you do though think about the implications on the testing, would you be able to do the same level of testing? Would having it on a function help you? If so can you bring the function to this file (under a different name please).

Reply:
Deploying go chaincode is likely to be reused in the all-in-one sample scenario so I would prefer to keep it out of this script